scripts/scil_tractogram_convert_trk_to_hdf5.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Save connections as TRK to HDF5.
+
+This script is useful to convert a set of connections or bundles to a single
+HDF5 file. The HDF5 file will contain a group for each input file, with the
+streamlines stored in the specified space and origin.
+
+To make a file compatible with scil_tractogram_commit.py or
+scil_connectivity_compute_matrices.py. you will have to follow this nomenclature
+for the input files:
+in_dir/
+    |-- LABEL1_LABEL1.trk
+    |-- LABEL1_LABEL2.trk
+    |-- [...]
+    |-- LABEL90_LABEL90.trk
+
+By default, ignores the empty connections. To save them, use the --save_empty.
+"""
+
+import argparse
+import logging
+import os
+
+from dipy.io.stateful_tractogram import Space, Origin
+from dipy.io.utils import is_header_compatible
+import h5py
+
+from scilpy.io.hdf5 import (construct_hdf5_header,
+                            construct_hdf5_group_from_streamlines)
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_overwrite_arg, add_verbose_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_bundles', nargs='+',
+                   help='Path of the input connections or bundles.')
+    p.add_argument('out_hdf5',
+                   help='Output HDF5 filename (.h5).')
```

Comment:
I had a question concerning this. In dwi_ml, I save my hdf5 files as .hdf5, not .h5. Any difference? Do we always accept them? Should we change the doc in all hdf5 scripts?

Reply:
There is no difference, when I looked at stuff done in Vanderbilt and Dipy, they used .h5 so I used the same
https://figshare.com/articles/dataset/DIPY_Synb0_weights/20277522/2